subsys/bluetooth/audio/tbs_client.c

```diff
@@ -110,26 +110,39 @@ static bool free_call_spot(struct bt_tbs_instance *inst)
 }
 #endif /* defined(CONFIG_BT_TBS_CLIENT_ORIGINATE_CALL) */
 
-static struct bt_tbs_instance *lookup_inst_by_handle(struct bt_conn *conn,
-							 uint16_t handle)
-{
-	uint8_t conn_index;
+struct find_inst_result {
 	struct bt_tbs_server_inst *srv_inst;
+	struct bt_tbs_instance *inst;
+};
 
-	__ASSERT(conn, "NULL conn");
+static int find_inst_by_handle(uint16_t handle, struct find_inst_result *result)
```

Comment:
This lookup function is invalid, as the handle is not unique per peer device

Reply:
Correct, you need to search for the `{handle, conn}` pair :)